src/district0x/ui/events.cljs

```diff
@@ -715,20 +758,24 @@
     nil))
 
 (reg-event-fx
-  :district0x.log/error
-  interceptors
-  (fn [{:keys [:db]} errors]
-    (apply console :error errors)
-    nil))
-
-(reg-event-fx
```

Comment:
use https://github.com/district0x/district-ui-logging#loginfo if possible

Reply:
I added the district.ui.logging library, but replacing everything is bigger refactoring job. I'll start another ticket in the meantime for future refactoring work.